packages/gasket-plugin-lint/lib/code-styles.js

```diff
@@ -204,11 +204,21 @@ const common = {
     if (hasEslint) {
       if (!pkg.has('scripts', 'lint')) {
         pkg.add('scripts', {
-          'lint': 'eslint --ext .js,.jsx,.cjs .',
+          'lint': `eslint --ext .js,.jsx,.cjs${typescript ? ',.ts,.tsx' : ''} .`,
```

Comment:
nit: it would be a little easier to digest this if it were split up some.

```suggestion
          'lint': `eslint --ext ${ext} .`,
```

with `ext` being something like:
```js
const ext = '.js,.jsx,.cjs' + (typescript ? ',.ts,.tsx' : '')
```